README.md

```diff
@@ -6,37 +6,37 @@
 
 Frugal is a hybrid, dynamic and static site generator that aims to minimize the amount of JavaScript served, thanks to partial hydration:
 
-- _Static pages rendered at build time_: by default Frugal produces static html.
-- _Server side pages_ render at request time
-- _Bring your own framework_: Frugal works with any UI framework able to compile to html
+- _Static pages rendered at build time_: by default Frugal produces static HTML
+- _Server side pages_ rendered at request time
+- _Bring your own framework_: Frugal works with any UI framework able to compile to HTML
 - _Manual partial hydration_ for interactive island in pages if you use Preact
 - _Client-side and server-side form submission_ for both static and dynamic pages
 - _Incremental build_: if both data and code did not change, the page is not rebuilt
 - _Client session_: transform your MPA into a SPA at little cost
 
 ## Getting Started
 
-Visit the [Getting started pages](https://frugal.deno.dev/doc@latest/getting-started) to get started with Frugal.
+Visit the [Getting started pages](https://frugal.deno.dev/doc@0.9/getting-started) to get started with Frugal.
```

Comment:
previously, it generates a 404 on `deno.dev`

Reply:
Fixed here : https://github.com/PaulBlanche/frugal/pull/68/commits/39e06adb4d68663a607c8af15a4c03876ce14a84 

typo on non capturing group, the `doc@latest/xxx` urls where not redirected to the latest existing version.